src/js/index.js

```diff
@@ -74,6 +91,45 @@ function App() {
     onModalClose();
     resetLotto();
   });
+
+  const issueManualLotto = () => {
+    const manualNumbers = document.querySelectorAll(".manual-number");
+    let manualNumberList = [];
+
+    manualNumbers.forEach((number) => {
+      manualNumberList.push(Number(number.value));
+    });
+
+    if (isValidPurchaseNumber(manualNumberList)) {
+      manualLottoNumberList = [];
+      for (let i = 0; i < manualNumberList.length / 6; i++) {
+        manualLottoNumberList[i] = manualNumberList.slice(i * 6, i * 6 + 6);
+      }
+    }
+    purchaseNewLottos();
+  };
+
+  $(".manual-purchase__form").addEventListener("submit", (e) => {
+    e.preventDefault();
+    const purchasePrice = $(".purchase__price-input").value;
+    manualLottoCount = $(".manual-purchase__price-input").value;
+    IsManualLotto = true;
+
+    if (!isValidPrice(purchasePrice)) {
+      return;
+    }
+
+    if (!isValidPurchaseAmount(manualLottoCount, purchasePrice)) {
+      return;
+    }
+
+    resetManualLotto();
+    addManualNumberInput(".manual-number-form", manualLottoCount);
+
+    $(".manual-purchase-btn").addEventListener("click", (e) => {
```

Comment:
- [ui-data-분리-참고문서](https://junilhwang.github.io/TIL/Javascript/Design/Vanilla-JS-Component/#_1-%E1%84%8F%E1%85%A5%E1%86%B7%E1%84%91%E1%85%A9%E1%84%82%E1%85%A5%E1%86%AB%E1%84%90%E1%85%B3%E1%84%8B%E1%85%AA-%E1%84%89%E1%85%A1%E1%86%BC%E1%84%90%E1%85%A2%E1%84%80%E1%85%AA%E1%86%AB%E1%84%85%E1%85%B5)
- 다른 분들 코드도 한 번 살펴봐서 data 중심으로 이벤트 -> data 변화 -> UI 변화로 코드를 가져보시면 좋을 것 같아요 

- e 안 쓰이면 안 적어주는 것이 좋겠죠 
- 이 이벤트는 처음에 폼에서 이벤트를 달아놓으면 구입할때 마다 이벤트 다는 코드가 실행되지 않아도 되겠죠 
```suggestion
    $(".manual-purchase-btn").addEventListener("click", () => {
```

Reply:
이상하게 e를 빼면 렌더링이 제대로 안되더라구요.. 어디선가 꼬인 것 같습니다.